### ISSUE TITLE
Please make a C version

### ISSUE THREAD

**Issue by muturikaranja** [JustasMasiulis/lazy_importer#42]
A lot of us would really like one..

**Comment by JustasMasiulis** [comment]
The problem with C is that AFAIK there are no ways to guarantee compile time computation.

I would probably be possible to create something with tons of macros, but 🤷

**Comment by muturikaranja** [comment]
Yeah one way is macros. Also, how can I compile this without the runtime?

**Comment by JustasMasiulis** [comment]
This doesn't really use any from "runtime". If you don't have C++ includes you can define `LAZY_IMPORTER_NO_CPP_FORWARD` to get rid of the only things I use from there.
